### PR TITLE
Fix Golangci-lint cache

### PR DESCRIPTION
**Pull request by Aoang**

### CODE REVIEW

**Comment by Aoang** [comment]
https://golangci-lint.run/usage/configuration/#cache

Here we can see the cache directory of golangci-lint.

**Comment by Aoang** [comment]
And more details https://pkg.go.dev/os#UserCacheDir https://github.com/golangci/golangci-lint/blob/19a338706a3dfae85ec5199216532860715fd269/internal/cache/default.go#L84

File: .github/workflows/commit.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ jobs:
         with:
           path: |
             ~/.cache/go-build
+            ~/.cache/golangci-lint
             ~/go/pkg/mod
             ~/go/bin
           key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
```
Comment (mathetake) [inline .github/workflows/commit.yaml:53]: could you change the key a bit to see if it's working correctly? 
```suggestion
          key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}-
```

Comment (mathetake) [reply .github/workflows/commit.yaml:53]: ah ok nvm! path change actually causes the invalidation

**Comment by codefromthecrypt** [comment]
thanks! also on your next commit can you sign it? ex. `git commit -asm"test cache"` this makes the DCO bot more happy

File: .github/workflows/commit.yaml [hunk 2]
```diff
@@ -47,9 +47,10 @@ jobs:
         with:
           path: |
             ~/.cache/go-build
+            ~/.cache/golangci-lint
             ~/go/pkg/mod
             ~/go/bin
-          key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
+          key: cache-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
```
Comment (mathetake) [inline .github/workflows/commit.yaml:53]: sorry the new cache has been saved successfully without key change: https://github.com/tetratelabs/wazero/runs/8242041868?check_suite_focus=true

could you revert this?

**Comment by mathetake** [comment]
Screen Shot 2022-09-08 at 13 50 32
cool it's working! awesome @Aoang

**Review by mathetake — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
Thanks for stopping by @Aoang and helping wazero! This stole minutes each PR!

**Pull request merged**